recipes/folly/all/conanfile.py

```diff
@@ -83,6 +83,12 @@ def _strip_options_requirements(self):
     def _required_boost_components(self):
         return ["context", "filesystem", "program_options", "regex", "system", "thread"]
 
+    # Freeze max. CMake version at 3.16.2 to fix the Linux build
+    def build_requirements(self):
+        if tools.os_info.is_linux and self.settings.os == "Linux":
+            if Version(self.version) <= "2020.08.10.00":
+                self.build_requires("cmake/3.16.2")
```

Comment:
```suggestion
        # TODO: Remove after Conan >=1.33
        self.build_requires("cmake/3.16.2")
```

Keep the same version for all compilers, so we will have reproducibility. The new Conan docker images are running Cmake 3.18, but they are not on CCI yet, but it should available on the next release,

Reply:
It can not surpase 3.18 https://github.com/conan-io/conan-center-index/pull/4048#discussion_r552133956

It seems to already be deployed based on the error and fixes

Reply:
You cannot remove this, not before you patched the folly again to build with current CMake version. see https://github.com/conan-io/conan-center-index/pull/4048#issuecomment-754626977

Reply:
Okay, let's keep 3.16. Is there any restriction?

Reply:
Not for the folly versions on CCI.